bundles/org.eclipse.swt/Eclipse SWT/cocoa/org/eclipse/swt/graphics/Image.java

```diff
@@ -77,6 +77,8 @@
  */
 public final class Image extends Resource implements Drawable {
 
+	private static final boolean USE_LEGACY_IMAGE_DISABLEMENT = Boolean.getBoolean("org.eclipse.swt.image.useLegacyDisablementAlgorithm");
```

Comment:
Should we make this property a bit shorter by just using the swt prefix (e.g. like for the `swt.autoScale` flag)? Or at least only `eclipse.swt...`?
```suggestion
	private static final boolean USE_LEGACY_IMAGE_DISABLEMENT = Boolean.getBoolean("swt.image.useLegacyDisablementAlgorithm");
```

Reply:
My impression was that every "public" flag uses the `org.eclipse.swt` prefix (like `org.eclipse.swt.browser.DefaultType`, `org.eclipse.swt.browser.IEVersion`, `org.eclipse.swt.browser.EdgeDataDir`, ...), not mentioning the varios `org.eclipse.swt.internal.*` flags. So for the sake of consistency, I used the `org.eclipse.swt` prefix here as well. The autoscale properties in `DPIUtil` seem to be the only place where a shorter prefix is used.

Still, this is only the reasoning behind the current choice, but I have no strong opinion on this. If we agree, I will gladly change to a shorter prefix.

Reply:
> My impression was that every "public" flag uses the org.eclipse.swt prefix

I think consistency is important so I'm fine with it, if that's the case. I just don't a full overview.